web-common-marlowe/src/Marlowe/Semantics.purs

```diff
@@ -1399,9 +1399,9 @@ addMoneyToAccount accId token amount accounts =
 {-| Gives the given amount of money to the given payee.
     Returns the appropriate effect and updated accounts
 -}
-giveMoney :: Payee -> Token -> BigInteger -> Accounts -> Tuple ReduceEffect Accounts
-giveMoney payee token@(Token cur tok) amount accounts = case payee of
-  Party party -> Tuple (ReduceWithPayment (Payment party (asset cur tok amount))) accounts
+giveMoney :: AccountId -> Payee -> Token -> BigInteger -> Accounts -> Tuple ReduceEffect Accounts
+giveMoney accountId payee token@(Token cur tok) amount accounts = case payee of
+  Party _ -> Tuple (ReduceWithPayment (Payment accountId payee (asset cur tok amount))) accounts
```

Comment:
before the party came from the case, and now it comes from the function definition... I wonder if that makes a difference 🤔

Reply:
This makes sense to me, since the case comes from the same parameter and now we are including the whole payee, not only the party. But see the other comment: it also affects the other side of the case